lyonza/WhiteTextBroadcast.py

```diff
@@ -0,0 +1,329 @@
+from pathlib import Path
+from re import findall
+
+from modules.CardType import CardType
+from modules.Debug import log
+from modules.RemoteFile import RemoteFile
+
+class WhiteTextBroadcast(CardType):
+    """
+    This class describes lyonza's CardType based on Wvdh's "WhiteTextBroadcast" card to show SxxExx format instead of absolute numbering
+    """
+
+    """Directory where all reference files used by this card are stored"""
+    REF_DIRECTORY = Path(__file__).parent.parent / 'ref'
+
+    """Characteristics for title splitting by this class"""
+    TITLE_CHARACTERISTICS = {
+        'max_line_width': 32,   # Character count to begin splitting titles
+        'max_line_count': 3,    # Maximum number of lines a title can take up
+        'top_heavy': False,     # This class uses bottom heavy titling
+    }
+
+    """Default font and text color for episode title text"""
+    TITLE_FONT = str(RemoteFile('lyonza', 'TerminalDosis-Bold.ttf'))
+    TITLE_COLOR = '#FFFFFF'
+
+    """Default characters to replace in the generic font"""
+    FONT_REPLACEMENTS = {'[': '(', ']': ')', '(': '[', ')': ']', '―': '-',
+                         '…': '...'}
+
+    """Whether this CardType uses season titles for archival purposes"""
+    USES_SEASON_TITLE = True
+
+    """Standard class has standard archive name"""
+    ARCHIVE_NAME = 'Broadcast Ordering Style'
+    
+    SEASON_TEXT_FORMAT = "S{season_number:02}"
+    
+    EPISODE_TEXT_FORMAT = "E{episode_number:02}"

```

Comment:
```suggestion
    EPISODE_TEXT_FORMAT = "S{season_number:02}E{episode_number:02}"
```
SEASON_TEXT_FORMAT is not used anywhere by the Maker